_tests/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM centos:7.3.1611
 MAINTAINER ihamilto@redhat.com
 
-RUN yum install -y  epel-release gcc gcc-c++ git libcurl-devel libffi-devel libtool libxml2 libxml2-devel libxslt \
+RUN yum clean all && yum install -y  epel-release gcc gcc-c++ git libcurl-devel libffi-devel libtool libxml2 libxml2-devel libxslt \
```

Comment:
:) Sorry, the clean should be the last command. The layered file system of Docker takes a snapshot of the filesystem at the end of each command and that becomes the new layer. You want those layers to be as small as possible when the command finishes. In this case, we'd want to clean up any yum caches, temp files, or downloaded files at the end of the run command so we only have on that layer what is absolutely necessary. I hope that makes more sense.